nucypher/control/controllers.py

```diff
@@ -312,6 +312,11 @@ def handle_request(self, method_name, control_request, *args, **kwargs) -> Respo
             request_body = control_request.data or dict()
             if request_body:
                 request_body = json.loads(request_body)
+
+            # handle querystring args from flask eg. ?quantity=11&duration_periods=100
+            if hasattr(control_request, 'args'):
+                request_body.update(control_request.args)
```

Comment:
If we go with the StringList field idea, this is all we would need and it would work with any request method.

Reply:
It is vital that these changes are merged ASAP!